app/models/recipients.rb

```diff
@@ -16,7 +16,16 @@ def user_emails
   end
 
   def users
-    User.joins(:roles).where( roles: {name: roles}).uniq
+    case group.to_sym
+    when :selected_teams
+      User.joins(:teams, :roles)
+        .where(teams: { kind: Team::KINDS }, roles: { name: roles }).uniq
```

Comment:
Do we need a season classifier? It will pull the currently selected teams for this year allright, but next year it will also include the almuni teams.

```patch
- .where(teams: { kind: Team::KINDS }, roles: { name: roles }).uniq
+ .where(teams: { kind: Team::KINDS, season: Season.current }, roles: { name: roles }).uniq
```

Or add another enum value that explicitly only selects the current year's students?